src/main/java/com/playmonumenta/networkrelay/RemotePlayerAPICommand.java

```diff
@@ -0,0 +1,35 @@
+package com.playmonumenta.networkrelay;
+
+import dev.jorel.commandapi.CommandAPICommand;
+import dev.jorel.commandapi.arguments.EntitySelectorArgument;
+import org.bukkit.command.CommandSender;
+
+public class RemotePlayerAPICommand {
+	public static void register() {
+		// base command
+		new CommandAPICommand("remoteplayerapi")
+			.withPermission("monumenta.networkrelay.remoteplayerapi")
+			.withSubcommand(new CommandAPICommand("get")
+				.withArguments(
+					new EntitySelectorArgument.OnePlayer("player")
+				)
+				.executes((sender, args) -> {
+					CommandSender player;
+					if (args[0] != null) {
+						player = (CommandSender) args[0];
+					} else {
+						player = sender;
+					}
+					// TODO: REMOVE THIS COMMAND, THIS IS FOR DEBUGGING DEV/MOD only - usb
```

Comment:
Since this is likely to be useful for development/debugging of anything using this API, it would be good to keep this, just restricted to people with the permission. Same suggestion for the argument type as the `/whereis` command.